aws-android-sdk-appsync/src/main/java/com/amazonaws/mobileconnectors/appsync/AWSAppSyncClient.java

```diff
@@ -259,9 +259,7 @@ private AWSAppSyncClient(AWSAppSyncClient.Builder builder) {
         subscriptionManager.setScalarTypeAdapters(new ScalarTypeAdapters(builder.customTypeAdapters));
         mS3ObjectManager = builder.mS3ObjectManager;
 
-        SubscriptionAuthorizer subscriptionAuthorizer = new SubscriptionAuthorizer(builder.mAwsConfiguration,
-            builder.mOidcAuthProvider,
-            applicationContext);
+        SubscriptionAuthorizer subscriptionAuthorizer = new SubscriptionAuthorizer(builder);
```

Comment:
`SubscriptionAuthorizer` needs the fields available on the builder to know if it needs to check the AWSConfiguration.